src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodLabel.java

```diff
@@ -112,5 +112,5 @@ public String toString() {
         public String getDisplayName() {
             return "Pod Label";
         }
-    }
+    }   
```

Comment:
Would you mind reverting unrelated whitespace changes? The patch to `src/main/` should be one line.